### ISSUE TITLE
THP (TRANSPARENT_HUGEPAGE) support RancherDesktop x86_84 vm kernel

### ISSUE THREAD

**Issue by schwichtgit** [rancher-sandbox/rancher-desktop#3191]
### Problem Description

I noted that the kernel offered by the Rancher Desktop VM differes in THP (linux TRANSPARENT_HUGEPAGE) support.
While the aarch64/arm64 (Apple Silicon) variant offers it, the x86_64 (Intel Mac) variant does not

`cat /sys/kernel/mm/transparent_hugepage/enabled`

a standard linux kernel would offer
`always [madvise] never`

RancherDesktop VM on MacOS for Apple Silicon has
`[always] madvise never`

while 
RancherDesktop VM on MacOS for Intel does not offer 
`/sys/kernel/mm/transparent_hugepage/enabled`
at all.

### Proposed Solution

at first I assumed that RancherDesktop is using a regular Alpine Linux and I discovered that Alpine Linux's virt kernel
misses the enablement in the kernel build config in x86_64 kernel config.

adding
`CONFIG_TRANSPARENT_HUGEPAGE=y`
to https://git.alpinelinux.org/aports/tree/main/linux-lts/virt.x86_64.config
should do the trick for alpine.

Today I opened an feature request issue for Alpine to address that: https://gitlab.alpinelinux.org/alpine/aports/-/issues/14272
I assume that there is additional work required by lima/RancherDesktop to benefit from a modifed virt kernel from Alpine.
(picking the new version etc)

**Comment by jandubois** [comment]
> I assume that there is additional work required by lima/RancherDesktop to benefit from a modifed virt kernel from Alpine.

Yes, but it will happen mostly automatically: we will pick up the new kernel automatically when we build new alpine-lima ISOs, but it is possible that it also means switching from 3.16 to 3.17. Feel free to ping here on this bug, or create a corresponding one directly in the alpine-lima repo once the change has landed upstream on Alpine (i.e. we will not create our own virt kernel, but will wait until upstream enables it).
